gateway/src/apicast/policy/content_caching/Readme.md

```diff
@@ -0,0 +1,50 @@
+# APICast Policy caching
+
+This policy allows to enable and disable caching based on customized conditions. These
```

Comment:
From a UX perspective, should we clarify that there are 2 ENVs to configure this policy and their default values?

Reply:
That is defined in the #upstream response codes.